wordpress-mobile/WordPress-Android
# [Home Page Picker] Hides preview button (Round 1)

**Fixes:** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2721

Depends on: https://github.com/wordpress-mobile/WordPress-Android/pull/13316

## Description

This PR hides the preview button. It should be reverted after round 1 (internal ref: pb3aDo-zl) of the Home Page Picker project.

## To test:

<details>
<summary><strong>To disable or enable the development version of Home Page Picker</strong></summary>

- Open the app from the build that allows FeatureFlags such as a PR build or a local development build
    - By default, the modal layout picker is enabled only in development builds.
    - From the site page:
        - Click on your Gravatar.
        - Click on App Settings.
        - Click on Test feature configuration.
        - Toggle "HomePagePickerFeatureConfig" to enable or disable the picker
        - Restart the App
</details>

Start the **Home Page Picker** screen:
1. Start the site creation flow (e.g. *Choose site > Add button*)
1. Select the *Create WordPress.com site* option
1. Select any design
1. Verify that only **Choose** button is visible at the bottom toolbar

PR submission checklist:
